apps/api/.env.example

```
@@ -16,11 +16,15 @@ GOLF_GENIUS_RETRY_DELAY_BASE=1000
- 
  # Mail configuration
 MAIL_FROM=noreply@example.com
 MAIL_HOST=smtp.example.com
 MAIL_PASS=your-password
 MAIL_PORT=587
-MAIL_USER=your-email@example.com
\ No newline at end of file
+MAIL_USER=your-email@example.com
+
+# Stripe configuration
+STRIPE_SECRET_KEY="sk_test_secret"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)